### DISCUSSION TITLE
Kroki in Asciidoc

### DISCUSSION THREAD

**Discussion by netthier** [requarks/wiki#6062] [Help / Questions] [answered]
I saw that Asciidoc support got added recently.
I started to translate my Markdown documents to Asciidoc, but I'm unsure how to use Kroki blocks.
In Markdown, I had the option to do this:
\```kroki
plantuml
@startuml
etc.
\```
to get a PlantUML diagram.
However, I'm unsure how to use this feature in Asciidoc documents.

**Comment by NGPixel** [comment, marked as answer]
Not all features are supported in AsciiDoc at the moment, Kroki being one of them.

**Reply by netthier** [reply]
Is there an ETA for Kroki support? I'd love to switch my wiki over to Asciidoc because of it's features compared to Markdown, but I use Kroki diagrams extensively.

**Reply by NGPixel** [reply]
AsciiDoc was a community contribution. Unless someone makes a PR to add kroki support, it won't be added until after v3 is released.
